pkg/client/platform_infrastructure/mock_client_service.go

```diff
@@ -0,0 +1,2285 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
```

Comment:
unless the code generator is added to the `make swagger` target, it will be wiped out by the cleanup done before generating the Go code: https://github.com/elastic/cloud-sdk-go/blob/1994e2808ea7279413637413e3243b2fa83a103e/build/Makefile.swagger#L45-L47.